components/ui/development/set-hostname.yaml

```
@@ -0,0 +1,4 @@
+---
+- op: add
+  path: /spec/host
+  value: @TBA
```

Kustomize build is failing because of the "@"
```suggestion
  value: TBA
```

---

I added quotes. I want to keep the @ sign since it would be useful later when adjusting the dev scripts.

---

You forgot to push the update?

---

I pushed the update now